modern-phytomorphology.csl

```diff
@@ -0,0 +1,167 @@
+      <key variable="issued"/>
+      <key variable="author"/>
+    </sort>
+    <layout prefix="(" suffix=")" delimiter="; ">
+      <group font-variant="small-caps" delimiter=" ">
+        <text macro="author-short" font-variant="normal"/>
+        <date variable="issued">
+          <date-part name="year"/>
+        </date>
+        <text variable="locator" prefix=": "/>        
+      </group>
+    </layout>
+  </citation>
+  <bibliography>
+    <sort>
+      <key macro="author-short"/>
+      <key variable="issued"/>
+    </sort>
+    <layout>
+      <text macro="author" suffix="." font-weight="bold"/>
+      <date variable="issued" prefix=" " suffix="." font-weight="bold">
+        <date-part name="year"/>
+      </date>
+      <choose>
+        <if type="bill book graphic legal_case legislation motion_picture report song" match="any">
+          <text macro="title" prefix=" "/>
+          <text macro="editor-translator" prefix=" "/>
+          <text prefix=" " suffix="." macro="publisher"/>
+          <text macro="access"/>
+        </if>
+        <else-if type="chapter" match="any">
+          <text macro="title" prefix=" " suffix="."/>
+          <group prefix=" In: ">
+            <names variable="editor translator">
+              <name font-variant="small-caps" delimiter-precedes-last="always" initialize-with="." name-as-sort-order="all" sort-separator=" "/>
+              <label form="short" prefix=" (" suffix="),"/>
+            </names>
+            <text variable="collection-title"/>
+            <text variable="container-title" prefix=" " suffix=": "/>
+            <text variable="page"/>
+          </group>
+          <text macro="publisher" prefix=" " suffix="."/>
+          <text macro="access"/>
+        </else-if>
+        <else-if type="paper-conference" match="any">
+          <text macro="title" prefix=" "/>
+          <group prefix=". " suffix=".">
+            <text variable="collection-title" prefix=" " suffix="."/>
+            <text variable="container-title" font-style="italic" prefix=" " suffix=": "/>
+            <text variable="page" suffix="."/>
+          </group>
+          <text macro="publisher" prefix=" " suffix=", "/>
+          <text macro="access"/>
+        </else-if>
+        <else-if type="webpage" match="any">
+          <text macro="title" font-variant="normal" prefix=" "/>
+          <group prefix=" " suffix=". ">
+            <text variable="collection-title" prefix=" " suffix="."/>
+            <text variable="container-title" prefix=" "/>
+          </group>
+          <text macro="publisher" prefix=" " suffix="."/>
+          <text macro="access"/>
+        </else-if>
+        <else>
+          <group suffix=".">
+            <text macro="title" prefix=" "/>
+            <text macro="editor-translator" prefix=" "/>
+          </group>
+          <group prefix=" " suffix=".">
+            <text variable="container-title" form="short" font-style="italic"/>
+            <group prefix=" ">
```

Comment:
no need for this group: just use `<text variable="volume" font-weight="bold" prefix=" "/>` and delete lines 158 and 160